repos/spack_repo/builtin/packages/darshan_runtime/package.py

```diff
@@ -17,7 +17,6 @@ class DarshanRuntime(AutotoolsPackage):
     systems where you intend to instrument MPI applications."""
 
     homepage = "https://www.mcs.anl.gov/research/projects/darshan/"
-    url = "https://web.cels.anl.gov/projects/darshan/releases/darshan-3.4.0.tar.gz"
```

Comment:
Instead of deleting this line, we should update the url property here to the new url format.